etl/steps/data/garden/AI/2023-06-07/monmoth_poll.meta.yml

```diff
@@ -0,0 +1,155 @@
+    services were provided by Braun Research, with sample obtained from Dynata (RDD,
+    n= 569), Aristotle (list, n= 152) and a panel of prior Monmouth poll participants
+    (n= 84). Monmouth is responsible for all aspects of the survey design, data weighting
+    and analysis. The full sample is weighted for region, age, education, gender and
+    race based on US Census information (ACS 2021 one-year survey). For results based
+    on this sample, one can say with 95% confidence that the error attributable to
+    sampling has a maximum margin of plus or minus 5.7 percentage points (adjusted
+    for sample design effects). Sampling error can be larger for sub-groups (see table
+    below). In addition to sampling error, one should bear in mind that question wording
+    and practical difficulties in conducting surveys can introduce error or bias into
+    the findings of opinion polls.
+  sources:
+  - name: Monmouth University Poll
+    description: The Monmouth University Poll was sponsored and conducted by the Monmouth
+      University Polling Institute from January 26 to 30, 2023 with a probability-based
+      national random sample of 805 adults age 18 and older. This includes 281 contacted
+      by a live interviewer on a landline telephone and 524 contacted by a live interviewer
+      on a cell phone, in English. Telephone numbers were selected through a mix of
+      random digit dialing and list-based sampling. Landline respondents were selected
+      with a modified Troldahl-Carter youngest adult household screen. Interviewing
+      services were provided by Braun Research, with sample obtained from Dynata (RDD,
+      n= 569), Aristotle (list, n= 152) and a panel of prior Monmouth poll participants
+      (n= 84). Monmouth is responsible for all aspects of the survey design, data
+      weighting and analysis. The full sample is weighted for region, age, education,
+      gender and race based on US Census information (ACS 2021 one-year survey). For
+      results based on this sample, one can say with 95% confidence that the error
+      attributable to sampling has a maximum margin of plus or minus 5.7 percentage
+      points (adjusted for sample design effects). Sampling error can be larger for
+      sub-groups (see table below). In addition to sampling error, one should bear
+      in mind that question wording and practical difficulties in conducting surveys
+      can introduce error or bias into the findings of opinion polls.
+    url: https://www.monmouth.edu/polling-institute/reports/monmouthpoll_us_021523/#:~:text=Nearly%203%20in%204%20(73,same%20as%20eight%20years%20ago.
+    source_data_url: https://www.monmouth.edu/polling-institute/documents/monmouthpoll_us_021523.pdf/
+    date_accessed: '2023-06-07'
+    publication_date: '2023-06-07'
+    publication_year: 2023
+    published_by: Monmouth (“Mon-muth”) University Poll, 2023
+tables:
+  monmoth_poll:
+    variables:
+      country:
```

Comment:
when I try to move this to grapher I keep getting an error - AssertionError: Column q1 must have a unit. It's weird because the q1 has a unit in the metadata.

Reply:
What exactly do you mean by moving this?

Reply:
well renaming the answer to country in the grapher step as I still need this hack at the end, right?